src/_scss/pages/agencyV2/visualizations/_totalObligationsOverTime.scss

```diff
@@ -43,6 +48,98 @@
                     }
                 }
             }
+            .budgetary-resources__item {
```

Comment:
Could we share code with the tooltip stying in `_barChart.scss`? Some of this looks like it is copied from the bar chart "hit box", which shouldn't be necessary here (the 20% width for example)

Reply:
Agreed, removed the container styling to use `bar-chart__item`. The other classes within are modified for the tooltip so I've left those in `_totalObligationsOverTime.scss`